CMakeLists.txt

```diff
@@ -309,7 +309,7 @@ if ( PLATFORM )
     set ( CPACK_PACKAGE_DESCRIPTION_SUMMARY "${PACKAGE_FILE_NAME_PREFIX}")
 else ( PLATFORM )
     if ( APPLE OR WIN32 )
-        set ( CPACK_PACKAGE_FILE_NAME "${PACKAGE_FILE_NAME_PREFIX}_${CPACK_RPM_PACKAGE_VERSION}-${version}-${stagever}${CPACK_SYSTEM_NAME}" )
+        set ( CPACK_PACKAGE_FILE_NAME "${PACKAGE_FILE_NAME_PREFIX}_${version}-${stagever}${CPACK_SYSTEM_NAME}" )
```

Comment:
Why do we do an underscore in the file name here instead of just a - like we use everywhere else?  The code looks fine btw, fixes the problem.  This is just more of a side question.